src/reader/TranslatableText.js

```diff
@@ -125,17 +145,26 @@ export function TranslatableText({
             pronouncing={pronouncing}
             translatedWords={translatedWords}
             setTranslatedWords={setTranslatedWords}
+            isBookmarkWord={isBookmarkWord}
           />
         );
       }
-      if (foundInstances[0] === word.id) {
+
+      // TODO: Consider moving this into exerciseTypes.isTranslation()
```

Comment:
// TODO: Consider moving this into a separate function named `isTranslationExercise(exerciseType)` inside of exerciseTypes